packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-cidr.yaml

```diff
@@ -0,0 +1,17 @@
+Resources:
+  CidrVpc1:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock:
+        !Cidr
+          - 192.168.1.1/24
+          - 2
+          - 5
+  CidrVpc2:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock:
+        !Cidr
+          - 192.168.1.1/24
```

Comment:
How about "92.168.1.1/24"?

Reply:
no problem I can do that, but can I ask why?